{"uuid":"94cb30c0-1d25-4b18-aa7b-b6777484c53e","historyId":"c59643030e1e776644fd5d1d6aa79cfc","status":"failed","statusDetails":{"message":"The following error originated from your test code, not from Cypress.\n\n  > Test argument \"title\" should be a string. Received type \"undefined\"\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.","trace":"TypeError: The following error originated from your test code, not from Cypress.\n\n  > Test argument \"title\" should be a string. Received type \"undefined\"\n\nWhen Cypress detects uncaught errors originating from your test code it will automatically fail the current test.\n\nCypress could not associate this error to any specific test.\n\nWe dynamically generated a new test to display this failure.\n    at createInvalidArgumentTypeError (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:104462:13)\n    at new Test (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:111251:11)\n    at context.it.context.specify (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:104662:18)\n    at http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:157496:21\n    at Object.create (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:104841:19)\n    at context.describe.context.context (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:104617:27)\n    at http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:157496:21\n    at http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:160053:16\n    at arrayEach (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:27108:11)\n    at Function.forEach (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:35988:14)\n    at evalScripts (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:160052:5)\n    at http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:160060:113\nFrom previous event:\n    at runScriptsFromUrls (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:160060:97)\n    at Object.runScripts (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:160074:10)\n    at $Cypress.onSpecWindow (http://www.webdriveruniversity.com/__cypress/runner/cypress_runner.js:149502:18)"},"stage":"finished","steps":[],"attachments":[{"name":"12-Traversal/traverse3.js.mp4","type":"video/mp4","source":"D:\\CyperssAutoation\\cypress\\videos/12-Traversal/traverse3.js.mp4"}],"parameters":[],"labels":[],"links":[],"start":1637305535398,"name":"An uncaught error was detected outside of a test","fullName":"An uncaught error was detected outside of a test","stop":1637305535424}